src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -48,4 +48,5 @@ Previously deprecated classes in the `spring-integation-hazelcast` module, such
 [[x7.0-mqtt-changes]]
 === MQTT Changes
 
-The `disconnectForcibly` method in `MqttPahoMessageDrivenChannelAdapter`, `Mqttv5PahoMessageDrivenChannelAdapter`, `Mqttv3ClientManager`, and `Mqttv5ClientManager` now supports `getQuiescentTimeout` in addition to the existing `disconnectCompletionTimeout`.
+The `disconnectForcibly` method in `MqttPahoMessageDrivenChannelAdapter`, `Mqttv5PahoMessageDrivenChannelAdapter`, `Mqttv3ClientManager`, and `Mqttv5ClientManager` now expose the option to configure `quiescentTimeout`, which is available through the https://eclipse.dev/paho/files/javadoc/org/eclipse/paho/client/mqttv3/IMqttAsyncClient.html[IMqttAsyncClient] client.
+See https://docs.spring.io/spring-integration/api/org/springframework/integration/mqtt/core/package-summary.html[MQTT Core Java Docs] for more information.
```

Comment:
This is not right.
We have talked about the target MQTT chapter of this reference manual, but you point to JavaDoc of the package.
It has to be like this:
```
See xref:mqtt.adoc[] for more information.
```

Reply:
Let's go with the final fix. We can revisit the docs later on, in that our javadocs cover the updates satisfactorly.